src/main/java/org/folio/circulation/domain/Item.java

```diff
@@ -178,7 +198,14 @@ public String getMaterialTypeName() {
   }
 
   public JsonArray getCopyNumbers() {
-    return getArrayProperty(getItem(), "copyNumbers");
+    JsonArray copyNumbers = getArrayProperty(getItem(), "copyNumbers");
+    return getEffectiveCopyNumbers(copyNumbers);
+  }
+
+  private JsonArray getEffectiveCopyNumbers(JsonArray copyNumbers) {
+    return copyNumbers.isEmpty()
+      ? copyNumbers.add(getProperty(holdingRepresentation, "copyNumber"))
+      : copyNumbers;
```

Comment:
Is it possible to have copyNumbers = null, not empty JsonArray?

Reply:
Calling `getArrayProperty(JsonObject representation, String propertyName)` method always returns a new JsonArray or its values. I assume that checking for null will be unnecessary.

Reply:
Feel free to add an overload for `getArrayProperty` that can define a default, if that helps with the comprehension of the code.

Reply:
Fix